service/src/main/kotlin/app/cash/backfila/ui/components/ToggleContainer.kt

```diff
@@ -0,0 +1,109 @@
+package app.cash.backfila.ui.components
+
+import kotlinx.html.ButtonType
+import kotlinx.html.TagConsumer
+import kotlinx.html.button
+import kotlinx.html.div
+import kotlinx.html.h2
+import kotlinx.html.id
+import kotlinx.html.section
+import kotlinx.html.span
+import misk.tailwind.icons.Heroicons
+import misk.tailwind.icons.heroicon
+
+fun TagConsumer<*>.ToggleContainer(
+  buttonText: String,
+  borderless: Boolean = false,
+  labelIsButton: Boolean = false,
+  isOpen: Boolean = false,
+  justifyBetween: Boolean = false,
+  labelBlock: TagConsumer<*>.() -> Unit = {},
+  hiddenBlock: TagConsumer<*>.() -> Unit,
+) {
+  val borderStyle = if (!borderless) "border-b border-t border-gray-200" else ""
+  section("grid items-center $borderStyle") {
+    attributes["data-controller"] = "toggle"
+
+    attributes["aria-labelledby"] = "info-heading"
+    h2("sr-only") {
+      id = "filter-heading"
+      +"""Filters"""
```

Comment:
Singe quotes?

Reply:
It's easier to leave it as triple if interpolation of variables is added later.